app/src/main/java/com/arctouch/codechallenge/util/MovieImageUrlBuilder.kt

```diff
@@ -1,17 +1,17 @@
 package com.arctouch.codechallenge.util
 
-import com.arctouch.codechallenge.api.TmdbApi
+import com.arctouch.codechallenge.BuildConfig
 
 private val POSTER_URL = "https://image.tmdb.org/t/p/w154"
 private val BACKDROP_URL = "https://image.tmdb.org/t/p/w780"
 
 class MovieImageUrlBuilder {
 
     fun buildPosterUrl(posterPath: String): String {
-        return POSTER_URL + posterPath + "?api_key=" + TmdbApi.API_KEY
+        return POSTER_URL + posterPath + "?api_key=" + BuildConfig.API_KEY
```

Comment:
Here you can get a better concise code through String templates of Kotlin:

    $POSTER_URL$posterPath?api_key=${BuildConfig.API_KEY}

Reply:
The same of const variables, I did not arrive to refactor this class, but I always use Strings templates in Kotlin.